src/shared/components/palette/address/write-address-field.component.ts

```diff
@@ -29,11 +30,16 @@ export class WriteAddressFieldComponent extends AbstractFieldWriteComponent impl
 
   addressOptions: AddressOption[];
 
-  missingPostcode = false;
+  alertService: AlertService;
 
-  constructor (addressesService: AddressesService, private isCompoundPipe: IsCompoundPipe) {
+
+  missingPostcode = false
+
```

Comment:
can we avoid multiple blank lines please?

Reply:
👍  done